pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -61,7 +54,8 @@ task qiime_import_from_bam {
         docker: docker
         memory: "${memory_mb} MiB"
         cpu: cpu
-        disks: "local-disk ${disk_size_gb} SSD"
+        disk: disk_size_gb + "GB"
+        disks: "local-disk" + disk_size_gb + "HDD"
```

Comment:
Actually -- this will also cause problems in Terra, you'll need a space between all these things
```suggestion
        disks: "local-disk " + disk_size_gb + " HDD"
```